lib/Core/Executor.cpp

```diff
@@ -3811,6 +3811,7 @@ void Executor::callExternalFunction(ExecutionState &state,
       assert(success && "FIXME: Unhandled solver failure");
       (void) success;
       ce->toMemory(&args[wordIndex]);
+      addConstraint(state, EqExpr::create(ce, *ai));
```

Comment:
Can you replace the solver call earlier to concretise the value with `toConstant(...)` like in https://github.com/klee/klee/blob/d79d048bfa21b432e1e9e97db9fca9924118b678/lib/Core/Executor.cpp#L4249

This handles the whole case more gracefully and adds the constraint as well..